src/WebJobs.Script/Binding/WebJobsCoreScriptBindingProvider.cs

```diff
@@ -139,14 +147,6 @@ public override Collection<Attribute> GetAttributes()
 
                 return attributes;
             }
-
-            override public Type DefaultType
-            {
-                get
-                {
-                    return typeof(Stream);
-                }
-            }
```

Comment:
Is the 'style issue' that they're not alphabetize? If so, I think we should turn off that rule. Code is IMO much more readable if members are organized in a logical way rather than alphabetically.